Sources/SectionScrubber.swift

```diff
@@ -109,30 +107,30 @@ public class SectionScrubber: UIView {
 
         super.init(frame: CGRectZero)
 
-        self.addSubview(scrubberImageView)
+        self.addSubview(self.scrubberImageView)
 
         self.setSectionlabelFrame()
         self.addSubview(self.sectionLabel)
 
         self.dragGestureRecognizer.addTarget(self, action: #selector(self.handleScrub))
         self.dragGestureRecognizer.delegate = self
+        self.addGestureRecognizer(self.dragGestureRecognizer)
 
         self.longPressGestureRecognizer.addTarget(self, action: #selector(self.handleScrub))
         self.longPressGestureRecognizer.minimumPressDuration = 0.2
         self.longPressGestureRecognizer.cancelsTouchesInView = false
         self.longPressGestureRecognizer.delegate = self
-
-        self.scrubberGestureView.addGestureRecognizer(self.longPressGestureRecognizer)
-        self.scrubberGestureView.addGestureRecognizer(self.dragGestureRecognizer)
-        self.addSubview(scrubberGestureView)
+        self.addGestureRecognizer(self.longPressGestureRecognizer)
```

Comment:
I added a scrubberGestureView because I find it strange behaviour if you can scrub on the left side of the scrubberView. It makes more sense if you can only scrub when you're pressing the scrubber thingy itself. But because this is just a bit to small to press comfortably I added this view of 44* width to be able to press the scrubber more intuitively.

I copied this behaviour from the google photo's app.

* from HIG: https://developer.apple.com/ios/human-interface-guidelines/visual-design/layout/